Mathlib/Algebra/Lie/Derivation/Basic.lean

```diff
@@ -390,13 +387,13 @@ end Inner
 
 section ExpNilpotent
 
-variable {R L : Type*} [Field R] [CharZero R] [LieRing L] [LieAlgebra R L] (D : LieDerivation R L L)
+variable {R L : Type*} [Field R] [LieRing L] [LieAlgebra R L] [LieAlgebra ℚ L]
```

Comment:
Now that we provide `[LieAlgebra ℚ L]` as extra data, we can weaken the assumptions on `R`:
```suggestion
variable {R L : Type*} [CommRing R] [LieRing L] [LieAlgebra R L] [LieAlgebra ℚ L]
```